salt/orchestrate/edx_sandbox/tag_ami.sls

```diff
@@ -0,0 +1,32 @@
+{% macro set_ami_status(image_id, ami_status) -%}
+set_status_for_edx_sandbox_ami_{{ image_id }}:
+  salt.function:
+    - name: saltutil.runner
+    - tgt: 'roles:master'
+    - tgt_type: grain
+    - arg:
+      - cloud.action
+    - kwargs:
+      - func: ec2.set_tags
+      - kwargs:
+        - resource_id: ami_id
+        - ami_status: {{ ami_status }}
+{%- endmacro %}
+
+{%
+  for image_id in salt['boto_ec2.find_images'](tags='{"ami_status": "active"}'):
```

Comment:
We should constrain this to also filter by the `role` tag so that, once we are building multiple different types of AMI, we don't accidentally set the wrong tags on multiple images.